x-pack/plugins/endpoint/public/applications/endpoint/components/page_view.tsx

```diff
@@ -20,24 +20,65 @@ import React, { memo, ReactNode } from 'react';
 import styled from 'styled-components';
```

Comment:
also just general question on our code organization: do we need to have a separate components folder? or does it make sense to just have all view related stuff just in the view folder? maybe common components can just be top level in the view folder? 🤷‍♀

Reply:
Thats what I was thinking - top level `components` folder was reusable across all views.
my approach also was that if a component from one view needs to be re-used in another View, we would elevate that component here for reuse - so that we're not importing from a sibling, only from a parent or a child of the given view/component.